### ISSUE TITLE
Importing not well-formed workflow gives no error

### ISSUE THREAD

**Issue by lpellegr** [ow2-proactive/studio#186]
If you try to import a not well-formed XML workflow definition, then nothing occurs. The operation ends as if no problem occurred. However, no Tasks are imported. An error message should be displayed to inform the user about the issue.

Below the workflow I used to try:

``` xml
<?xml version="1.0" encoding="UTF-8"?>
<job
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xmlns="urn:proactive:jobdescriptor:3.2"
     xsi:schemaLocation="urn:proactive:jobdescriptor:3.2 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.2/schedulerjob.xsd"
    name="Untitled workflow"
    priority="normal"
    cancelJobOnError="false">
    <oops oops oops
  <description>
    <![CDATA[ A workflow that executes python in JVM. ]]>
  </description>
  <taskFlow>
    <task name="Python_Task">
      <description>
        <![CDATA[ The simplest task, ran by a python engine. ]]>
      </description>
      <scriptExecutable>
        <script>
          <code language="python">
            <![CDATA[
for x in range(1, 11):
    print x
]]>
          </code>
        </script>
      </scriptExecutable>
    </task>
  </taskFlow>
</job>
```

**Comment by lpellegr** [comment]
The issue is still present with the current master. Using option _Import as current workflow_ or _Add to current workflow_ makes no difference.

**Comment by amouhoub** [comment]
After investigating this issue, it turns out that the server returns an error message and stacktrace that normally should be printed out in the message area to inform the user. However, because the JSON error message contains tags, it causes the navigator's JSON.parse() to throw an exception because it is not HTML-encoded.
A possible fix is to encode the error message on the server for this case in particular (invalid xml job file). A client-side fix is to force extract the "errorMessage" from the "unparsable" JSON using regex. It would do fine until the JSON model of the server's response is modified.

**Comment by amouhoub** [comment]
This issue is also valid for scheduling-portal, and the aforementioned merge solves this problem for scheduling-portal but the issue is still open for "studio"

**Comment by amouhoub** [comment]
Fixed in PRs #479 #481 and #485
